src/components/CollapsedOpeningTimes.js

```diff
@@ -74,7 +87,10 @@ export default class CollapsedOpeningTimes extends React.Component {
         ];
 
         let openingHours = _(this.props.object.openingTimes)
-            .sortBy(record => order.indexOf(record.day));
+            .sortBy(record =>
+                (order.indexOf(record.day) * 24) +
+                (parseInt(record.open))
```

Comment:
What is `record.open`? Is it like `"01:00:00"`? If so, `parseInt` would always just give `1`